packages/stainless/src/stl.ts

```diff
@@ -123,6 +163,10 @@ export type GetEndpointMethod<E extends AnyEndpoint> =
 export type GetEndpointUrl<E extends AnyEndpoint> =
   E["endpoint"] extends `${HttpMethod} ${infer Url}` ? Url : never;
 
+/**
+ * Gets all endpoints associated with a given resource
+ */
+// TODO make this a method on AnyResourceConfig?
```

Comment:
🤔 @jedwards1211 wdyt?

Reply:
we could, I'm not sure it would be a big win right now for the amount of work to convert `Resource` to a class

Reply:
Hmm we could make it a property with an anonymous function that captures the object, but that's a bit hacky

Reply:
nah, we should just update the comment to clarify that it's blocked on moving Resource to a class, which we may want to do at some point anyway. For now this method is totally fine, and we could even just remove the TODO comment.